spec/models/event_spec.rb

```diff
@@ -34,15 +36,21 @@
   end
 
   describe '#resolve' do
-    it 'enqueues job' do
+    let(:project) { create :project }
+    before do
+      subject.name = 'comment'
+      subject.data = { pull_request: { id: 1000 } }
+    end
+
+    it 'creates a event' do
       expect {
-        described_class.resolve(event: '')
-      }.to have_enqueued_job(EventJob)
+        subject.resolve
+      }.to change(Event, :count).by(1)
     end
 
     it 'calls event class' do
-      expect(Events::PullRequest).to receive(:resolve).with(event: 'pull_request')
-      described_class.resolve(event: 'pull_request')
+      expect(subject).to receive(:resolve)
```

Comment:
I don't understand this test. Isn't it trivial?

You expect `subject` to receive `resolve` and then call `subject.resolve`.

Reply:
Yes! My bad